addons/udes_suggest_location/tests/test_suggest_exactly_match_move_line.py

```diff
@@ -0,0 +1,68 @@
+from odoo.exceptions import ValidationError
+from . import common
+from ..registry.suggest_locations_policy import SUGGEST_LOCATION_REGISTRY
+
+
+class TestSuggestExactlyMatchMoveLine(common.SuggestedLocations):
+    @classmethod
+    def setUpClass(cls):
+        super(TestSuggestExactlyMatchMoveLine, cls).setUpClass()
+        policy_name = "exactly_match_move_line"
+        cls.ByProduct = SUGGEST_LOCATION_REGISTRY[policy_name](cls.env)
+        cls.picking_type_pick.u_suggest_locations_policy = policy_name
+
+        cls.create_quant(cls.apple.id, cls.stock_location.id, 10)
+
+        cls._pick_info = [{"product": cls.apple, "uom_qty": 5}]
+
+    def test_correct_location_suggested_for_generic_location_destination(self):
+        mls = self._generate_move_lines_for_picking_for_given_destination_location(
+            self.out_location.id
+        )
+
+        self._check_move_lines_destination_location_match_expected_location(
+            mls, self.out_location
+        )
+
+
+    def test_correct_location_suggested_for_view_location_destination(self):
```

Comment:
I'm not sure what is expected. It looks like if its a view it returns an empty record set of locations and then because its empty nothing changes from default on the move line? But i think this is because i haven't looked recently at this dir. But if you extend the doc string in the comment above i think it will help me :)

Reply:
Yep that is how that works. It is how it is implemented in UDES11, it does feel pointless. I thought it might be implemented this way so its clear what is happening when selecting a policy in the config.

Reply:
I changed the unit test to check that the suggested location returns an empty 'stock.location' recordset, I think this is a better test for the suggest_locations() function.